File: Python/백준/Silver/2563. 색종이/색종이.py
N = int(input())
paper = [[0] * 100 for _ in range(100)]

for _ in range(N):
    x, y = map(int, input().split())
    for i in range(x, x + 10):
        for j in range(y, y + 10):
            paper[i][j] = 1  # 칸에 색종이가 붙었음을 표시

total = 0
for i in range(100):
    total += sum(paper[i])
print(total)

File: Java/백준/Silver/2563. 색종이/색종이.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int N = sc.nextInt();
        int[][] paper = new int[100][100];

        for (int k = 0; k < N; k++) {
            int x = sc.nextInt();
            int y = sc.nextInt();
            for (int i = x; i < x + 10; i++) {
                for (int j = y; j < y + 10; j++) {
                    paper[i][j] = 1;
                }
            }
        }
        int total = 0;
        for (int i = 0; i < 100; i++) {
            for (int j = 0; j < 100; j++) {
                total += paper[i][j];
            }
        }
        System.out.println(total);
        sc.close();
    }
}
